example_code/java-date-example/src/DateExamples.java

```diff
@@ -0,0 +1,32 @@
+import java.time.Instant;
+import java.util.Date;
+
+public class DateExamples {
+
+    private Date date;
+
+    public void constructors() {
+        Date date1 = new Date();
+    }
+
+    public void staticMethods() {
+        Instant instant = Instant.now();
+        Date date1 = Date.from(instant);
+    }
+
+    public void instanceMethods() {
+        Date date1 = new Date();
+        Date date2 = new Date();
+
```

Comment:
Add examples for `new Date(long)` and `new Date(String)`
```suggestion

        Date dateFromLong = new Date(123456L);
        Date dateFromString  = new Date("somedate");
        
```